libs/transactions/src/lib/extensions/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './transaction-addition-details';
```

Comment:
I don't see the approach with an internal index files a lot, would it make sense to avoid using it, to make it consistent? Or opposite, use it then everywhere.